pages/services/hdfs/2.5.0-2.6.0-cdh5.11.0/release-notes/index.md

```diff
@@ -0,0 +1,108 @@
+---
+layout: layout.pug
+navigationTitle: Release Notes
+excerpt: Release notes for v. 2.5.0-2.6.0-cdh5.11.0
+title: Release Notes
+menuWeight: 10
+model: /services/hdfs/data.yml
+render: mustache
+---
+## Version 2.5.0-2.6.0-cdh5.11.0
```

Comment:
Let's remove dots at the end of sentences in bullet lists - sometimes they're present, sometimes not. Prior release docs don't have them

Reply:
Updated.